convert-hf-to-gguf.py

```diff
@@ -74,9 +77,13 @@ def __init__(self, dir_model: Path, ftype: gguf.LlamaFileType, fname_out: Path,
         self.lazy = not eager
         self.part_names = Model.get_model_part_names(self.dir_model, ".safetensors")
         self.is_safetensors = len(self.part_names) > 0
+        
         if not self.is_safetensors:
             self.part_names = Model.get_model_part_names(self.dir_model, ".bin")
+            
         self.hparams = Model.load_hparams(self.dir_model)
+        self.gguf_writer = gguf.GGUFManager(fname_out, gguf.MODEL_ARCH_NAMES[self.model_arch], split_arguments,
+                                            endianess=self.endianess, use_temp_file=self.use_temp_file)
```

Comment:
This needs to be lower in `__init__`, or else `fname_out` won't always be correct, especially with outtype templating.

Reply:
Fixed, thank you. The placement caught my eye in the diffs but I didn't realize it was important.